docs/volumegroupreplication.md

```diff
@@ -0,0 +1,37 @@
+# VolumeGroupReplication
+
+VolumeGroupReplication is a namespaced resource that contains references to storage object to be grouped and replicated, VolumeGroupReplicationClass corresponding to the driver providing replication, VolumeGroupContent and VolumeReplication CRs.
+
+`volumeGroupReplicationClassName` is the name of the class providing group replication.
+
+`volumeReplicationClassName` is the name of the class providing the replication for volumeReplication CR.
+
+`volumeReplicationName` is the name of the volumeReplication CR created by this volumeGroupReplication CR
+
+`volumeGroupReplicationContentName` is the name of the volumeGroupReplicationConten CR created by this volumeGroupReplication CR.
+
+`replicationState` is the state of the volume being referenced. Possible values are `primary`, `secondary` and `resync`.
```

Comment:
"volume" -> "volume group", here and in more places

Reply:
Done